### PR TITLE
Remove dead code causing warnings

### PR DESCRIPTION
**Pull request by pedrocr**

### CODE REVIEW

File: src/platform/linux/x11/window.rs [hunk 1]
```diff
@@ -144,45 +136,11 @@ impl XWindow {
...
     }
 }
 
-#[derive(Clone)]
-pub struct WindowProxy {
-    data: Arc<Mutex<Option<WindowProxyData>>>,
-}
-
-impl WindowProxy {
-    pub fn wakeup_event_loop(&self) {
```
Comment (tomaka) [inline src/platform/linux/x11/window.rs]: It's really suspicious that this is a dead code.

Comment (pedrocr) [reply src/platform/linux/x11/window.rs]: It definitely is right now but maybe it shouldn't be, no idea.

Comment (pedrocr) [reply src/platform/linux/x11/window.rs]: reading some code it seems EventsLoopProxy is implemented with different code and this is just a left over from when there was a WindowProxy in the API.

**Comment by tomaka** [comment]
Thanks

**Pull request merged**